mantle/platform/platform.go

```diff
@@ -555,3 +555,12 @@ func WriteJSONInfo(m Machine, w io.Writer) error {
 	e.SetIndent("", "")
 	return e.Encode(info)
 }
+
+// Create a temporary log file
+func CreateTempLogFile(builder *QemuBuilder) string {
```

Comment:
Hmm, I don't think this warrants a function. Let's just directly call `builder.TempFile` from the test code.